assignments/objects.js

```diff
@@ -49,16 +91,40 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: "Susan",
+  age: 70,
+  speak: function() {
+    console.log( "My name is " + this.name + "!")
```

Comment:
you may want to use a 'return' instead of a console.log in this function, so that later when you call the function it returns something.